src/utils/sanityClient.ts

```diff
@@ -1,11 +1,11 @@
 import sanityClient from "@sanity/client";
-import config from "../../sanity/sanity.json";
 import imageUrlBuilder from "@sanity/image-url";
 import {SanityImageSource} from "@sanity/image-url/lib/types/types";
+import {erProd} from "./restUtils";
 
 const client = sanityClient({
     projectId: "hvfvg2j3",
-    dataset: "production",
+    dataset: erProd() ? "production" : "test",
```

Comment:
Er det mulig å sjekke på `erDev()` og defaulte til `production` isteden? 
Slik at dersom konfigurasjonen i fremtiden skulle være feil/vi bytter url etc. , så vil `default produksjon` føre til at feilsituasjonen kun påvirker test og ikke prod :)

Reply:
Brukte `erProd()` av ren latskap, siden de andre `erX()` funksjonene våre var spesifikke for hvert miljø. Kan eventuelt bytte ut til `!erProd() ? "test": "production`? 

Ved omskriving til nextjs er planen uansett å trekke ut mye av dette til env-variabler, siden vi ikke har tilgang til `window`-objektet på serversiden. :)

Reply:
!erProd() gir ikke så mye mer 😄 

Men generelt er det lurt å ha defaulten til production. 
Det avhenger veldig av hvor stor konsekvens det har dersom dev-config havner i prod